public/pages/Dashboard/Components/AnomaliesLiveChart.tsx

```diff
@@ -96,14 +96,22 @@ export const AnomaliesLiveChart = (props: AnomaliesLiveChartProps) => {
   const getLiveAnomalyResults = async () => {
     setIsLoadingAnomalies(true);
     // check if there is any anomaly result in last 30mins
-    const latestSingleLiveAnomalyResult = await getLatestAnomalyResultsByTimeRange(
-      searchES,
-      '30m',
-      dispatch,
-      -1,
-      1,
-      true
-    );
+    // need to initially check if there is an error when accessing anomaly results index
+    // in the case that it doesn't exist upon cluster initialization
+    let latestSingleLiveAnomalyResult = [] as any[];
+    try {
+      latestSingleLiveAnomalyResult = await getLatestAnomalyResultsByTimeRange(
+        searchES,
+        '30m',
+        dispatch,
+        -1,
+        1,
+        true
```

Comment:
what if you change this to `false`?

Reply:
wouldn't that change the functionality of the call? I think we just want to encapsulate any error here (which can definitely be caused by the index not existing yet) and set is loading to false.